### DISCUSSION TITLE
How to set up the webhooks for dynamic updating

### DISCUSSION THREAD

**Discussion by jnegrete2005** [abraunegg/onedrive#2885] [Q&A] [answered]
Hey! So I have been all day trying to set up my valid HTTPS certificate with chatgpt, and I can't get it to work. I am using Manjaro. Can someone send me a tutorial to set this up? I have no idea about the subject so I don't know what to search.

**Comment by abraunegg** [comment, marked as answer]
@jnegrete2005 

> Can someone send me a tutorial to set this up? I have no idea about the subject so I don't know what to search.

Have you read the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/webhooks.md ?

This is pretty much a walk through tutorial on *how* to set it up .....

Important note - the instructions are very specific and you must have all other items/equipment to make this work. If you do not have your own router|firewall to perform the port forwarding change, this is never going to work for you. If your ISP blocks all inbound traffic to you (some do) this will also never work for you.

It is well beyond the scope of this site to walk you through any of that beyond what has been documented.

Please seek support from Manjaro Support Folk if you are getting stuck.

**Reply by jnegrete2005** [reply]
I understand that the webhooks tutorial sets you up for the onedrive automatic updates. What I am looking for is getting a resolvable https url. I was trying, but it isn't resolvable from the internet. I will look support from Manjaro wiki. Thanks.

**Reply by abraunegg** [reply]
As per the documentation:

![image](https://github.com/user-attachments/assets/8e2061a2-0d96-4cdd-9222-00a3d22acb8c)

Please read|research these organisations on how they can provide you with a resolvable HTTPS URL.
